### PR TITLE
feat(plasma-*): Add .npmignore and remove file field for test files

### PR DESCRIPTION
**Pull request by neretin-trike**
Разница в размере между тарниками из актуальных версий пакетов и канареек. Различие в несколько байт скорее всего обусловлено изменением исходного кода package.json

image

:baby_chick: Download canary assets:

[colors--canary.322.4125774581.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.322.4125774581.xml)  
[color_metro_ios-swift--canary.322.4125774581.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.322.4125774581.swift)  
[color_metro_kotlin--canary.322.4125774581.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.322.4125774581.kt)  
[color_metro_react-native--canary.322.4125774581.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.322.4125774581.ts)  
[color_sbermarket_ios-swift--canary.322.4125774581.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.322.4125774581.swift)  
[color_sbermarket_kotlin--canary.322.4125774581.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.322.4125774581.kt)  
[color_sbermarket_react-native--canary.322.4125774581.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.322.4125774581.ts)  
[color_sberprime_ios-swift--canary.322.4125774581.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.322.4125774581.swift)  
[color_sberprime_kotlin--canary.322.4125774581.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.322.4125774581.kt)  
[color_sberprime_react-native--canary.322.4125774581.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.322.4125774581.ts)  
[color_selgros_ios-swift--canary.322.4125774581.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.322.4125774581.swift)  
[color_selgros_kotlin--canary.322.4125774581.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.322.4125774581.kt)  
[color_selgros_react-native--canary.322.4125774581.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.322.4125774581.ts)  
[color_smbusiness_ios-swift--canary.322.4125774581.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.322.4125774581.swift)  
[color_smbusiness_kotlin--canary.322.4125774581.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.322.4125774581.kt)  
[color_smbusiness_react-native--canary.322.4125774581.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.322.4125774581.ts)  
[PlasmaTokensColor--canary.322.4125774581.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.322.4125774581.swift)  
[shadow_sbermarket_react-native--canary.322.4125774581.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.322.4125774581.ts)  
[typo_mage_ios-swift--canary.322.4125774581.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.322.4125774581.swift)  
[typo_mage_kotlin--canary.322.4125774581.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.322.4125774581.kt)  
[typo_mage_react-native--canary.322.4125774581.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.322.4125774581.ts)  
[typo_plasma_ios-swift--canary.322.4125774581.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.322.4125774581.swift)  
[typo_plasma_kotlin--canary.322.4125774581.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.322.4125774581.kt)  
[typo_plasma_react-native--canary.322.4125774581.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.322.4125774581.ts)  
[typo_ruler_ios-swift--canary.322.4125774581.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.322.4125774581.swift)  
[typo_ruler_kotlin--canary.322.4125774581.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.322.4125774581.kt)  
[typo_ruler_react-native--canary.322.4125774581.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.322.4125774581.ts)  
[typo_sage_ios-swift--canary.322.4125774581.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.322.4125774581.swift)  
[typo_sage_kotlin--canary.322.4125774581.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.322.4125774581.kt)  
[typo_sage_react-native--canary.322.4125774581.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.322.4125774581.ts)  
[typo_sbermarket_ios-swift--canary.322.4125774581.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.322.4125774581.swift)  
[typo_sbermarket_kotlin--canary.322.4125774581.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.322.4125774581.kt)  
[typo_sbermarket_react-native--canary.322.4125774581.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.322.4125774581.ts)  
[typo_soulmate_ios-swift--canary.322.4125774581.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.322.4125774581.swift)  
[typo_soulmate_kotlin--canary.322.4125774581.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.322.4125774581.kt)  
[typo_soulmate_react-native--canary.322.4125774581.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.322.4125774581.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.136.0-canary.322.4125774581.0
  npm install @salutejs/plasma-core@1.93.0-canary.322.4125774581.0
  npm install @salutejs/plasma-icons@1.121.0-canary.322.4125774581.0
  npm install @salutejs/plasma-temple@1.130.0-canary.322.4125774581.0
  npm install @salutejs/plasma-ui@1.163.0-canary.322.4125774581.0
  npm install @salutejs/plasma-web@1.162.0-canary.322.4125774581.0
  npm install @salutejs/plasma-sb-utils@0.91.0-canary.322.4125774581.0
  # or 
  yarn add @salutejs/plasma-b2c@1.136.0-canary.322.4125774581.0
  yarn add @salutejs/plasma-core@1.93.0-canary.322.4125774581.0
  yarn add @salutejs/plasma-icons@1.121.0-canary.322.4125774581.0
  yarn add @salutejs/plasma-temple@1.130.0-canary.322.4125774581.0
  yarn add @salutejs/plasma-ui@1.163.0-canary.322.4125774581.0
  yarn add @salutejs/plasma-web@1.162.0-canary.322.4125774581.0
  yarn add @salutejs/plasma-sb-utils@0.91.0-canary.322.4125774581.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-322/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-322/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-322/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-322/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-322/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-322/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-322/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-322/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-322/temple-storybook/

**Comment by Yeti-or** [comment]
@Yakutoc проверь пожалуйста локально

**Comment by Yakutoc** [comment]
@Yeti-or Проверил локально - работает.

File: packages/plasma-b2c/.npmignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/components/**/*.component-test*
+/es/components/**/*.component-test*
+src
```
Comment (Yeti-or) [inline packages/plasma-b2c/.npmignore:3]: но оно и так не кладется, зачем исключать?

storybook / package-lock.json / jest.config.js / tsconfig.json

File: packages/plasma-b2c/.npmignore [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+.storybook
+/package-lock.json
+/index.d.ts.map
```
Comment (Yeti-or) [inline packages/plasma-b2c/.npmignore:6]: а ты это проверил? в итоге не нужно для навигации?

File: packages/plasma-b2c/.npmignore (hunk 1 above)
Comment (neretin-trike) [reply packages/plasma-b2c/.npmignore:3]: на самом деле всё это кладётся, т.к. я полностью удалил file из package.lock, который имеет приоритет над .npmignore. Поэтому вместо вайт листа, пришлось делать точечный блэк лист

File: packages/plasma-b2c/.npmignore (hunk 2 above)
Comment (neretin-trike) [reply packages/plasma-b2c/.npmignore:6]: предлагаю индекс пока не трогать (потому что он не генерируется, а определен нами заранее), а во всех остальных случаях да, не нужен

File: packages/plasma-b2c/.npmignore (hunk 1 above)
Comment (Yeti-or) [reply packages/plasma-b2c/.npmignore:3]: стоп подожди, а почему ты file то полностью удалил? они вместе что до сих пор не работают?

File: packages/plasma-b2c/package.json [hunk 3]
```diff
@@ -91,23 +88,6 @@
...
         "typescript-coverage-report": "0.7.0"
     },
-    "keywords": [
-        "design-system",
-        "react-components",
-        "ui-kit",
-        "react"
-    ],
-    "files": [
-        "components",
```
Comment (Yeti-or) [inline packages/plasma-b2c/package.json:101]: `components/**/*.ts?x`

**Review by Yeti-or — APPROVED** [review]

**Pull request merged**